### ISSUE TITLE
Better styling for callout boxes

### ISSUE THREAD

**Issue by topepo** [aml4td/website#4]
I don't like the quarto default: 

<img width="606" alt="image" src="https://github.com/aml4td/website/assets/5731043/7b8fee04-370d-4699-b2b9-dbe66538fe28">

The ones in _Interactively exploring high-dimensional data and models in R_ ([examples](https://dicook.github.io/mulgar_book/4-pca.html)) are beautiful and Mine's [class notes](https://openintro-ims2.netlify.app/17-inference-two-props) ones are also an improvement.

**Comment by krz** [comment]
What do you have in mind?
Something with an image like in your first link? Which colors?

here is something you can try in [https://jsfiddle.net/](https://jsfiddle.net/)
![image](https://github.com/aml4td/website/assets/210623/370ca4c6-ab6a-4468-b127-a3bf404f9659)

html:
```
<div class="insight">
Some content.
</div>
```

css:
```
.insight {
  padding: 2em;
  padding-left: 100px;
  margin: 2em 0;
  background-color: #e2ecb0;
  background-image: url("https://i.ibb.co/zVFnfq1/leaf-clean.png");
  background-size: 60px;
  background-position: 10px 1em;
  background-repeat: no-repeat;
  border-right: 6px solid #5b6d3c;
  position: relative;
  font-size: 0.9em;
  border-radius: 3px;
}
```

**Comment by topepo** [comment]
Looking around, something like [these](https://dribbble.com/shots/12883156-Callout-component-Ui-Figma-React-design-system) would be good. I like the solid color in the header and body along with the bevel.

**Comment by krz** [comment]
maybe something like this:
![image](https://github.com/aml4td/website/assets/210623/5605b5b6-646c-42f8-9529-56214ef1109a)

html:
```
<div class="importantinfo">
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
</div>
```

css:
```
@import url('https://fonts.googleapis.com/css2?family=Libre+Franklin:wght@100;300&display=swap');

.importantinfo {
  padding: 0.25em 1em 1em 3em;
  margin-top: 20px;
  margin-bottom: 20px;
  background: #a9d3ab 5px center/3em no-repeat;
  background-image: url("https://svgshare.com/i/11W1.svg");
  background-position: 0.6em 0.5em;
  background-size: 3% auto;
  border-radius: 10px;
  font-size: 18px;
  font-family: "Libre Franklin", sans-serif;
}


.importantinfo::before {
  content: "Important information";
  font-family: "Libre Franklin", sans-serif;
  font-weight: bolder;
  display: block;
  padding: 0.4em 0em;
  vertical-align: middle;
  font-size: 20px;
}
```

**Comment by topepo** [comment]
That looks great!

**Comment by topepo** [comment]
We might be able to use fontawesome or iconify images (free collections).

**Comment by topepo** [comment]
[These](https://github.com/microsoft/fluentui-system-icons/blob/main/icons_filled.md) icons are under a MIT license.

**Comment by topepo** [comment]
Can you make a PR into `aml4td.scss` with this and a [GH url](https://raw.githubusercontent.com/microsoft/fluentui-system-icons/086c7fdf41e88c8d18cf7edf6c42a471cf47b36f/assets/Important/SVG/ic_fluent_important_32_filled.svg)? 

In the quarto sources, we can use this syntax: 

```
::: {.importantinfo}
Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua.
:::
```

It requires the devel version of Quarto. 

We really appreciate your help.

**Comment by krz** [comment]
nice, thanks. This is how it looks:

![image](https://github.com/aml4td/website/assets/210623/aa811b46-f10d-4b8e-b4fd-3514eb0302ac)

Will open a PR later and work on some different color options

**Comment by topepo** [comment]
Closed via ##28
